content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -26,18 +26,17 @@ You can use either `kubectl create configmap` or a ConfigMap generator in `kusto
 
 ### Create a ConfigMap Using kubectl create configmap
 
-Use the `kubectl create configmap` command to create configmaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
+Use the `kubectl create configmap` command to create ConfigMaps from [directories](#create-configmaps-from-directories), [files](#create-configmaps-from-files), or [literal values](#create-configmaps-from-literal-values):
 
 ```shell
 kubectl create configmap <map-name> <data-source>
 ```
 
-where \<map-name> is the name you want to assign to the ConfigMap and \<data-source> is the directory, file, or literal value to draw the data from.
+Where \<map-name> is the name you want to assign to the ConfigMap and \<data-source> is the directory, file, or key and literal value to draw the data from.
```

Comment:
don't change the 'where' word

Reply:
why add 'key and'?

Reply:
@tengqm Thanks for the review and catching that. I have made the suggested changes.